docs/markdown/snippets/cpp-debug.md

```diff
@@ -0,0 +1,10 @@
+## `debug` optimization type handles C++ stdlib assertions
+
+The `debug` release type now passes preprocessor defines to enable
```

Comment:
release type? Optimization type? It's the build type, right.

But this is conditional on the ndebug setting (which has an if-release mode), not the debug setting.